wp-content/themes/core/pcss/admin/editor-style.pcss

```diff
@@ -78,6 +58,26 @@ body {
 @import "../base/forms/controls/textarea";
 @import "../base/forms/validation/required";
 
+/* -----------------------------------------------------------------------------
```

Comment:
The latest versions of postcss-partial-import require imports to come before any other css declarations (like regular css requires). Hence, adjusting the order here.